doc/source/examining/loading_data.rst

```diff
@@ -2112,6 +2112,78 @@ For example, to add support for a longint field named
 ``my_custom_field``, one would add ``('my_custom_field', 'l')`` to ``assoc``.
 
 
+Particle ages and formation times
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For cosmological simulations that include star particles, RAMSES stores particle
+formation times as conformal times. To access the formation time field data in
+conformal units use the ``"conformal_birth_time"`` field. This will return
+the formation times of particles in the simulation in conformal units as a
+dimensionless array. To access the formation time in physical units, use the
+``"particle_birth_time"`` field. Finally, to access the ages of star
+particles in your simulation, use the ``"star_age"`` field. Note that this field
+is defined for all particle types but will only make sense for star particles.
+
+For non-cosmological simulations, the particle ages are already in physical
+units on disk, so there will not be a ``"conformal_birth_time"`` field. To
+access the formation time in physical units, use the ``"particle_birth_time"``
+field.
+
+For all simulations, the time is equal to zero at the beginning of the
```

Comment:
The time is equal to zero at the beginning of the simulation for ideal simulations. Instead, for cosmological simulations, it is equal to zero at z=0 i.e. the end of the simulation